apps/backoffice/src/components/app-preview/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Close, Info, InfoOutlined, Title } from "@mui/icons-material";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  Stack,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
```

Comment:
Should clear unused import

Reply:
[f4d7e91](https://github.com/pagopa/io-services-cms/pull/810/commits/f4d7e910ff98cdc289b4e2e1a6304f42a3afeecf)